nix/modules/tests/kld.nix

```diff
@@ -62,6 +62,7 @@
     db1.wait_until_succeeds("curl -s http://127.0.0.1:9273/metrics")
 
     db1.wait_until_succeeds("kld-cli get-info")
+    db1.wait_until_succeeds("kld-cli lsp-list-protocols")
```

Comment:
Please remove this from here. Its not the right place to test the API. This get-info call is only to check if KLD has started. You can add a call to list-protocols in the smoke test if you like.